src/routes/CreateTrip.jsx

```diff
@@ -1,75 +1,120 @@
+      localStorage.setItem('trips', JSON.stringify(storedTrips));
 
-        navigate(`/trip-details/${response}`)
+      // Отмечаем форму как успешно отправленную
+      setIsSubmitted(true);
     }
+  };
 
-
-
-
-    return (
-        <>
-            <div className='flex  flex-col p-10  pt-20 sm:m-10'>
-                <h1 className='text-4xl font-bold font-sans self-center'>Create New Trip</h1>
-                <p className=' text-sm sm:text-base pt-8 self-center'>Fill in the details to start a new trip</p>
-                <div className='flex flex-col sm:flex-row justify-around pt-10  sm:gap-20 gap-12'>
-                    <div className='flex flex-col'>
-                        <h3 className='sm:text-xl text-lg font-medium pb-5'>Trip Name</h3>
-                        <input type="text" value={title} onChange={handleChange} className='text-2xl font-bold pl-4 sm:w-20w w-full  p-1 h-12 h-10 border-2 border-black rounded-2xl ' />
-                    </div>
-                    <div className='flex flex-col'>
-                        <h3 className='sm:text-xl text-lg font-medium pb-5'>Currency</h3>
-                        <div className='flex flex-row sm:gap-10 gap-6 flex-wrap'>
-                            <button className='border-2 border-greygrey  p-2 rounded-xl transition hover:bg-black hover:text-white font-medium  hover:-translate-y-0.5 hover:scale-105'>$ CAD</button>
-                            <button className='border-2 border-greygrey  p-2 rounded-xl transition hover:bg-black hover:text-white font-medium  hover:-translate-y-0.5 hover:scale-105'>$ USD</button>
-                            <button className='border-2 border-greygrey  p-2 rounded-xl transition hover:bg-black hover:text-white font-medium  hover:-translate-y-0.5 hover:scale-105'>€ Euro</button>
-                            <button className='border-2 border-greygrey  p-2 rounded-xl transition hover:bg-black hover:text-white font-medium  hover:-translate-y-0.5 hover:scale-105'>£ GBP</button>
-                        </div>
-                    </div>
-                </div>
-                <div className='self-center pt-20 sm:mt-20'>
-                    <button onClick={createTrip} className={valid ? 'sm:w-20w self-center text-center text-lg font-medium shadow-lg border rounded-lg border-black py-2 px-20 text-white bg-black transition hover:bg-white hover:text-black hover:-translate-y-1 hover:scale-105'
-                        : 'sm:w-20w self-center text-center text-lg font-medium shadow-lg border rounded-lg border-black py-2 px-20 text-gray-600 bg-zinc-300'}>Start Trip</button>
-                </div>
-            </div>
-        </>
-    );
+  return (
+    <div className="sm:px-20 pt-1 sm:m-20 min-h-screen">
+      <h1 className="text-4xl font-bold text-center">Create a New Trip</h1>
+      {isSubmitted ? (
+        <div className="mt-8 text-center">
+          <h2 className="text-2xl font-bold text-green-600">Trip successfully created!</h2>
```

Comment:
Submitting the request does not confirm the trip's creation; instead, receiving a status code 200 from the trip creation request does.